### PR TITLE
Updated FE quality tools docs (manual reviews & unit testing)

### PR DESCRIPTION
**Pull request by kristinagore**
## Description
Added "Manual review" page and updated "Writing a new unit test" and "Unit testing"

## Testing done
Updates reviewed by SMEs

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Changes have been tested in vets-website
- [ ] Changes have been tested in IE11, if applicable
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx [hunk 1]
```diff
@@ -118,26 +133,134 @@ fillDate(
 
 >- `selectCheckbox()` - Enzyme helper that fires a change event with a value for a checkbox at the given name
 
+## Example unit tests
+
+VSP provides helper utilities to make writing tests easier. The following example unit tests illustrate some of these helpers. You can find these unit tests at `src/applications/vic-v2/tests/config/veteranInformation.unit.spec.jsx`:
```
Comment (billfienberg) [inline packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx]: There is no `src/applications/vic-v2/tests/config/veteranInformation.unit.spec.jsx` in this repo (`veteran-facing-services-tools`). 

Here is the location of that file in the `vets-website` repo: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vic-v2/tests/config/veteranInformation.unit.spec.jsx

File: packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx [hunk 2]
```diff
@@ -118,26 +133,134 @@ fillDate(
...
+import formConfig from '../config/form.js';
+
+describe('VIC veteran information', () => {
+  const { schema, uiSchema } = formConfig.chapters.veteranInformation.pages.veteranInformation;
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data=\{{}}
```
Comment (billfienberg) [inline packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx]: I could be wrong, but I suspect that `\` character is not supposed to be there. 

```
- data=\{{}}
+ data={{}}
```

File: packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx (hunk 1 above)
Comment (kristinagore) [reply packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx]: I made that change. Thanks for finding!

File: packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx [hunk 3]
```diff
@@ -118,26 +133,134 @@ fillDate(
...
+
+    expect(form.find('input').length).to.equal(7);
+    expect(form.find('select').length).to.equal(4);
+  });
+
+  ...
+});
+```
+
+Helpers are imported from `schemaform-utils.jsx`. The `DefinitionTester` is a component you can use to simulate a page being rendered without having to set up a whole form application with all the dependencies.
```
Comment (billfienberg) [inline packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx]: The `schemaform-utils.jsx` file actually lives in the `vets-website` repo. 

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/unit/schemaform-utils.jsx

File: packages/documentation/src/pages/platform/front-end-standards/manual-reviews.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+VSP reviews logs to Sentry to ensure that any new logs are useful and do not contain personally identifiable information (PII).
+
+Examples:
+- Do not log an entire request response if all that's important is an error code.
+- Do not log PII such as names, emails, or user IDs.
+
+### ESLint
+
+VSP enforces [ESLint rules](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/engineering/frontend/eslint) through CircleCI to enforce code quality. ESLint rules are included in the `.eslintrc` file, which is located in the root folder. Any rules that VSP is [testing for inclusion](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/frontend/eslint/adding-new-rules.md) are located in the `circle.eslint.json` file.
```
Comment (billfienberg) [inline packages/documentation/src/pages/platform/front-end-standards/manual-reviews.mdx]: `circle.eslint.json` lives in the `vets-website` repo:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/circle.eslint.json

Comment (kristinagore) [reply packages/documentation/src/pages/platform/front-end-standards/manual-reviews.mdx]: Added link

File: packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx (hunk 3 above)
Comment (kristinagore) [reply packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx]: Added link

File: packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx (hunk 2 above)
Comment (kristinagore) [reply packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx]: Removed

**Pull request merged**